### PR TITLE
Fixes discrepancies in Admin Sales Fee Summary

### LINKED ISSUE #5707: Admin Sales: Discrepancies in Admin Sales' Fee Summary
**Describe the bug**

Currently, `Tickets` column contains amount of sales by `Completed`, `Pending` and `Expired` orders. Also, `maximum_fee` is not returned.
Also, the `Revenue` doesn't includes `maximum_fee` into calculation.

**To Reproduce**
1. Go to `Admin`
2. Click on `Sales`
3. Click on `Fees`

**Expected behavior**

`Tickets` should have number of `Completed` orders only. Also, `maximum_fee` must be included in calculation.

**Additional context**

Working on it. Related frontend issue - https://github.com/fossasia/open-event-frontend/issues/2318

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #5707 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Currently, Tickets column contains amount of sales by Completed, Pending and Expired orders. Also, maximum_fee is not returned.
Also, the Revenue doesn't includes maximum_fee into calculation.

#### Changes proposed in this pull request:
- Tickets should have number of Completed orders only. 
- `maximum_fee` must be included in calculation.

### CODE REVIEW

File: app/api/admin_sales/fees.py [hunk 1]
```diff
@@ -22,13 +22,14 @@ class Meta:
     name = fields.String()
     payment_currency = fields.String()
     fee_percentage = fields.Float(attribute='fee')
+    maximum_fee = fields.Float(attribute='maximum_fee')
```
Comment (iamareebjamal) [inline app/api/admin_sales/fees.py]: Why?

Comment (shreyanshdwivedi) [inline app/api/admin_sales/fees.py]: @iamareebjamal this variable is included to provide info on the frontend regarding max fees.
Screenshot_2019-03-23 Fees Sales Administration Open Event(2)

**Comment by shreyanshdwivedi** [comment]
@iamareebjamal @CosmicCoder96 please review

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 please review

**Pull request merged**